### DISCUSSION TITLE
System (GPU) requirements

### DISCUSSION THREAD

**Discussion by Feodor0090** [ppy/osu#19540] [Q&A]
Which GPU capabilities are minimal to run the game? OpenGL version, shader model, OGL extensions, etc.

I'm looking for as much cheap GPU as possible, so wanted to know how far can I go. I couldn't find this information anywhere.

**Comment by peppy** [comment]
We don't have any specific requirement as they are a moving target.

If you care about running perfectly *today* then avoid intel GPUs from 5 years or more, and make sure you understand NVIDIA optimus will not let you push 10,000 frames.
